indico/web/flask/session.py

```diff
@@ -22,6 +23,10 @@
 from indico.web.util import get_request_user
 
 
+with open('indico/moment-locales.yml') as f:
```

Comment:
That looks somewhat fragile... I'd rather doing it during the first request (the function can be memoized with `@functools.cache` to avoid parsing it more than once) with an absolute path (and .yaml since we already use that for other yaml files):

```py
        with open(os.path.join(current_app.root_path, 'moment-locales.yaml')) as f:
            ...
```

(or the pathlib equivalent ;))